boards/st/nucleo_h723zg/nucleo_h723zg.dts

```diff
@@ -190,3 +191,36 @@ zephyr_udc0: &usbotg_hs {
 &rng {
 	status = "okay";
 };
+
+&flash0 {
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		/* 128KB for bootloader */
+		boot_partition: partition@0 {
+			label = "mcuboot";
+			reg = <0x00000000 DT_SIZE_K(128)>;
+			read-only;
+		};
+
+		/* primary application image slot: 384KB */
+		slot0_partition: partition@20000 {
+			label = "image-0";
+			reg = <0x00020000 DT_SIZE_K(384)>;
+		};
+
+		/* secondary application image slot: 384KB */
+		slot1_partition: partition@80000 {
+			label = "image-1";
+			reg = <0x00080000 DT_SIZE_K(384)>;
+		};
```

Comment:
Hi @thomasbouston, thanks for contributing.
slot0 should be larger than slot1, see discussion here https://github.com/zephyrproject-rtos/zephyr/pull/71376#discussion_r1560743472